.github/workflows/build_and_test.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         run: |
           touch ${{ env.LOG_DIR}}/local-setup.log
           ./local-setup/launch.py local-setup/github-action-config.json 2>&1 | tee ${{ env.LOG_DIR}}/local-setup.log &
-          sleep 60
+          sleep 120
```

Comment:
Had to increase the time between launching local setup and running the test scripts, since in the local setup launching, we already sleep for 60s between starting workers. So until now, worker 2 was often not even running when we executed the test scripts.